app/templates/components/forms/reset-password-form.hbs

```diff
@@ -20,7 +20,46 @@
             {{#if token}}
               <div class="ui left icon input">
                 <i class="key icon"></i>
-                {{input type='password' name='password' placeholder=(t 'password') value=password}}
+                {{#if showNewPass}}
+                  {{input type='text' name='password' placeholder=(t 'New Password') value=password}}
+                {{else}}
+                  {{input type='password' name='password' placeholder=(t 'New Password') value=password}}
+                {{/if}}
+                <div class="ui small basic icon buttons">
+                  <button type="button" class="ui button" {{action 'showNewPassword'}}>
+                    {{#if showNewPass}}
```

Comment:
A lot of duplication in here. Please reduce it and make the code optimal

Reply:
Then there will be no space in between both the textfields, thats why repeated the else part

Reply:
![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/35539313/74866676-41e69480-5379-11ea-9cef-5f48c31c3012.gif)

Reply:
It is implemented similar to that in sign up part

Reply:
> Goal is to improve the code as we move on. "It was implemented this way" doesn't really work

Remove duplication as much as possible